notebooks/community/model_garden/model_garden_jax_vision_transformer.ipynb

```diff
@@ -0,0 +1,855 @@
+{
```

Comment:
<p>Explain what these functions do</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1997/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>in cell 'Define common functions'</p>

Reply:
Done.